File: TicTacToe/TicTacToe.Shared/Dto/NewMoveDto.cs
namespace TicTacToe.Api.Shared.Dto
{
    public class NewMoveDto
    {
        public int PlayerId { get; set; }

        public int GameId { get; set; }

        public int PositionX { get; set; }

        public int PositionY { get; set; }
    }
}

File: TicTacToe/TicTacToe.Angular/src/app/data/dto/new-move-dto.ts
export class NewMoveDto{
    PlayerId: number = 0;
    GameId: number = 0;
    PositionX: number = 0;
    PositionY: number =0;

    constructor(PlayerId: number, GameId: number, PositionX: number, PositionY: number){
        this.PlayerId = PlayerId;
        this.GameId = GameId;
        this.PositionX = PositionX;
        this.PositionY = PositionY;
    }
}